diff -urN StepMania-3.9-src.org/src/GameState.h StepMania-3.9-src/src/GameState.h
--- StepMania-3.9-src.org/src/GameState.h	2007-03-19 11:41:07.000000000 +0100
+++ StepMania-3.9-src/src/GameState.h	2007-03-19 11:41:34.000000000 +0100
@@ -85,8 +85,8 @@
 	bool AnyPlayersAreCpu() const;
 
 	void GetCharacters( vector<Character*> &apCharactersOut );
-	Character* GameState::GetRandomCharacter();
-	Character* GameState::GetDefaultCharacter();
+	Character* GetRandomCharacter();
+	Character* GetDefaultCharacter();
 
 	PlayerController	m_PlayerController[NUM_PLAYERS];
 	
